Applications/FileIO/GocadIO/GocadAsciiReader.h

```diff
@@ -26,31 +26,44 @@ namespace FileIO
 {
 namespace Gocad
 {
-class GocadTSurfaceReader final
+
+enum class GOCAD_DATA_TYPE
```

Comment:
```suggestion
enum class GocadDataType
```
Similar to the enums in MeshLib, for example: MeshElemType, CellType, MeshQualityType, etc.

Reply:
:heavy_check_mark: